src/model/exceptions/throttled-rundown-exception.ts

```diff
@@ -0,0 +1,8 @@
+import { ErrorCode } from '../enums/error-code'
+import { Exception } from './exception'
+
+export class ThrottledRundownException extends Exception {
+  constructor(message: string) {
+    super(ErrorCode.RUNDOWN_IS_LOCKED, message)
```

Comment:
The error is still `RUNDOWN_IS_LOCKED` though i see it wasn't created by you in this PR. 
From what i can see this is the only place it is used so maybe that should be changed to fit the naming as well :-)

Reply:
That is not create in this PR. I can change it, but can we have confirmation to that? @LindvedKrvang @KvelaGorrrrnio
I can rename it to:  `RUNDOWN_IS_THROTTLET`

Reply:
Confirming that we can change it to `RUNDOWN_IS_THROTTLED` :)